README.md

```diff
@@ -51,31 +51,31 @@ Requirements: MonoGame + Visual Studio
 
 1. Download and install **Visual Studio 2017** (2015 is not supported) and **NET Framework 4.6**
 
-2. Clone whole repository:
+2. Clone the whole repository:
 
 `git clone https://github.com/MaKiPL/OpenVIII.git`
 
-3. Download and install **development build** of MonoGame:
+3. Download and install the **development build** of MonoGame:
 [MonoGame for Visual Studio](http://teamcity.monogame.net/repository/download/MonoGame_PackagingWindows/latest.lastSuccessful/MonoGameSetup.exe?guest=1)
 
-4. If you get "Unable to load DLL 'FreeImage'", Download and install:
+4. If you get an "Unable to load DLL 'FreeImage'" error, download and install:
 [Visual C++ Redistributable Packages for Visual Studio 2013](https://www.microsoft.com/en-us/download/details.aspx?id=40784)
 
-5. In Visual Studio 2017, while the solution is open, goto Tools > NuGet Package Manager > Manage NuGet Packages for solution. Make sure the required packages are installed and everything is up to date. There should be a notice on this screen if a package isn't installed. There will be a number in a box next to Updates if there are out of date packages.
+5. In Visual Studio 2017, while the solution is open, go to Tools > NuGet Package Manager > Manage NuGet Packages for solution. Make sure the required packages are installed and everything is up to date. There should be a notice on this screen if a package isn't installed. There will be a number in a box next to Updates if there are out of date packages.
 
-6. Make sure you add Final Fantasy VIII path to array at `WindowsGameLocationProvider.cs:36`. On Windows the code tries to detect the install path via the registry. If it fails, it'll fall back to the array.
+6. Make sure you add the Final Fantasy VIII path to the array at `WindowsGameLocationProvider.cs:36`. On Windows the code tries to detect the install path via the registry. If it fails, it'll fall back to the array.
 
 7. That's all. You can now compile the executable.
 
 ## Getting started (Linux/Mono) [Tested on Ubuntu]
 
-1. Make sure your Linux is up to date. Due to FFmpeg dependency we require Ubuntu Cosmos
+1. Make sure your Linux is up to date. Due to the FFmpeg dependency we require Ubuntu Cosmos.
```

Comment:
"Due to ffmpeg dependencies, we require Ubuntu Cosmos."

Reply:
But it's a single dependency on the ffmpeg library, not multiple dependencies.